scapy/arch/windows/__init__.py

```diff
@@ -72,11 +72,40 @@ def _encapsulate_admin(cmd):
     # rights, which will execute the command
     return "Start-Process PowerShell -windowstyle hidden -Wait -Verb RunAs -ArgumentList '-command &{%s}'" % cmd
 
+def _hack_windows_subprocess(r=100):
```

Comment:
I know it's a hack, but can it be given more descriptive method name?
Something along the lines of "_suppress_file_handles_inheritance"?

Reply:
I agree.

Reply:
Good idea. Updated